### PR TITLE
Spanish Translation: Copyable/Editable/AddRemove/SearchBar/PreviewNav

### PR DESCRIPTION
**Pull request by oruburos**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named

### CODE REVIEW

**Comment by oruburos** [comment]
@andrewn Can you have a look at this, please?

File: client/components/AddRemoveButton.jsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import { withTranslation } from 'react-i18next';
+
 
 import AddIcon from '../images/plus.svg';
 import RemoveIcon from '../images/minus.svg';
 
-const AddRemoveButton = ({ type, onClick }) => {
-  const alt = type === 'add' ? 'Add to collection' : 'Remove from collection';
+const AddRemoveButton = ({ type, onClick, t }) => {
+  const alt = type === 'add' ? t('AddRemoveButton.altAdd') : t('AddRemoveButton.altRemove');
```
Comment (andrewn) [inline client/components/AddRemoveButton.jsx:10]: This is an ARIA label, so I'd expect this to follow the naming conventions for ARIA labels.

File: client/components/PreviewNav.jsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+        <LogoIcon role="img" aria-label={t('Common.p5logoARIA')} focusable="false" className="svg__logo" />
       </div>
       <Link className="nav__item" to={`/${owner.username}/sketches/${project.id}`}>{project.name}</Link>
-      <p className="toolbar__project-owner">by</p>
+      <p className="toolbar__project-owner">{t('PreviewNav.ByUser')}</p>
       <Link className="nav__item" to={`/${owner.username}/sketches/`}>{owner.username}</Link>
     </div>
     <div className="nav__items-right">
-      <Link to={`/${owner.username}/sketches/${project.id}`} aria-label="Edit Sketch" >
+      <Link to={`/${owner.username}/sketches/${project.id}`} aria-label={t('PreviewNav.EditSketch')} >
```
Comment (andrewn) [inline client/components/PreviewNav.jsx:20]: Append "ARIA" to match naming convention?

File: client/modules/IDE/components/EditableInput.jsx [hunk 3]
```diff
@@ -58,7 +59,7 @@ function EditableInput({
       <button
         className="editable-input__label"
         onClick={beginEditing}
-        aria-label={`Edit ${displayValue} value`}
+        aria-label={this.props.t('EditableInput.EditValue', { display: displayValue })}
```
Comment (andrewn) [inline client/modules/IDE/components/EditableInput.jsx:62]: The variable will be named `display` but the translation text in english uses `{{ displayValue }}` and in spanish `{{ display }}`. Please fix.

**Review by andrewn — CHANGES_REQUESTED** [review]
A couple of minor naming things, then this is good to go!

File: client/components/AddRemoveButton.jsx (hunk 1 above)
Comment (oruburos) [reply client/components/AddRemoveButton.jsx:10]: ok, done

File: client/components/PreviewNav.jsx (hunk 2 above)
Comment (oruburos) [reply client/components/PreviewNav.jsx:20]: you're right, it will be corrected.

File: client/modules/IDE/components/EditableInput.jsx (hunk 3 above)
Comment (oruburos) [reply client/modules/IDE/components/EditableInput.jsx:63]: yes, it should be ```display``` in both. It's fixed now.

**Comment by oruburos** [comment]
@andrewn The issues were addressed, can you have a look, please?

**Review by andrewn — APPROVED** [review]
Perfect, thanks!

**Comment by andrewn** [comment]
@oruburos can you fix the conflict and then I'll merge. Thanks!

**Comment by oruburos** [comment]
@andrewn Conflicts solved, can you have a look at this, please?

**Review by andrewn — APPROVED** [review]

**Pull request merged**